F/FMUComplianceChecker/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "FMUComplianceChecker"
+version = v"2.0.4"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/modelica-tools/FMUComplianceChecker/releases/download/2.0.4/FMUChecker-2.0.4-linux64.zip", "02f6d1a175fe4c51d5840ef40fcd05ca7fb3ceec170d7825d9c946d75eae12eb"),
```

Comment:
```suggestion
    ArchiveSource("https://github.com/modelica-tools/FMUComplianceChecker/releases/download/$(version)/FMUChecker-$(version)-linux64.zip", "02f6d1a175fe4c51d5840ef40fcd05ca7fb3ceec170d7825d9c946d75eae12eb"),
```

Reply:
I don't think this binary would work on a Musl-based system